lab_2_retrieval_w_bm25/main.py

```diff
@@ -18,6 +19,13 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned.
     """
+    if not isinstance(text, str):
+        return None
+    for symbol in text:
+        if not symbol.isalpha() and not symbol in ["-", " ","'"]:
```

Comment:
better use tuple and create it before loop

Reply:
`not symbol in ["-", " ","'"]` is mistake probably . Use `symbol not in ["-", " ","'"]`

Reply:
done